server/src/queues/unlinkDatabase.ts

```diff
@@ -0,0 +1,80 @@
+import { dbTypeToDokkuPlugin } from './../graphql/utils';
+import { Database, App } from '../generated/graphql';
+import { Worker, Queue } from 'bullmq';
+import createDebug from 'debug';
+import { pubsub } from './../index';
+
+import { config } from '../config';
+import { sshConnect } from '../lib/ssh';
+import { prisma } from '../prisma';
+
+const queueName = 'unlink-database';
+const debug = createDebug(`queue:${queueName}`);
+
+interface QueueArgs {
+  app: App;
+  database: Database;
+}
+
+export const unlinkDatabaseQueue = new Queue<QueueArgs>(queueName, {
+  defaultJobOptions: {
+    // Max timeout 20 minutes
+    timeout: 1.2e6,
+  },
+  connection: config.redisClient,
+});
+
+/**
+ * - Unlink db
+ */
+const worker = new Worker(
+  queueName,
+  async (job) => {
+    const { app, database } = job.data;
+    debug(
+      `starting unlinkDatabaseQueue for ${database.type} database ${database.name} from  ${app.name} app`
+    );
+    console.log(
```

Comment:
let's remove all the console.log before merging :)

Reply:
@pradel done.